src/components/user/uc/users.uc.unit.test.js

```diff
@@ -99,6 +148,8 @@ describe('users usecase', () => {
 			const result = await userUC.deleteUser(USER_ID, 'student', { account: currentUser });
 
 			expect(result.userId).to.deep.equal(USER_ID);
+			expect(result.data.filter((d) => d.scope === 'user')[0].data).to.be.not.undefined;
```

Comment:
in order to not blow this test up too much, I would prefer seperate unit tests that focus on pseudonym/registration pin specifically